### DISCUSSION TITLE
FinRL and High Frequency Trading

### DISCUSSION THREAD

**Discussion by vmohl** [AI4Finance-Foundation/FinRL#739] [Q&A]
Hi,

which parts of FinRL would have to be changed to use it on Limit Order Book Data, instead of minute, hourly or daily data? Did somebody already work with that?

Thank you in advance for any help.

**Comment by kami-ayati** [comment]
you can use list length or list size for limit order book .
and it is not instead of time frames.

**Comment by zhumingpassional** [comment]
the data layer including the calc of indicators, and env should be changed.

**Reply by shibanatifkhann** [reply]
How did you change the env?

**Comment by kami-ayati** [comment]
Good
Best

**Comment by kami-ayati** [comment]
Hi
Online on pdf help and platform.
Best
